### PR TITLE
QAE-50 - E2E test case for footer-builder-margin

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added E2E test case for footer builder margin.

### Screenshots

https://share.getcloudapp.com/o0uZNy40

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/footer-builder/margin.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/footer-builder/margin.test.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/customize';
+import { setBrowserViewport } from '../../../utils/set-browser-viewport';
+import { scrollToElement } from '../../../utils/scroll-to-element';
+describe( 'footer builder margin setting in customizer', () => {
+	it( 'margin should apply correctly', async () => {
+		const footerBuildermargin = {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/margin.test.js:7]: The constant variable must be declared as camelCase.

File: tests/e2e/specs/customizer/footer-builder/margin.test.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+			'section-footer-builder-layout-margin': {
+				mobile: {
+					top: '40',
+					right: '40',
+					bottom: '40',
+					left: '40',
+				},
+				'mobile-unit': 'px',
+			},
+			'footer-desktop-items': {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/footer-builder/margin.test.js:115]: footer-desktop-item code needs to run a margin test case or we can remove this extra code, check once?

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/margin.test.js:115]: it's needed for footer builder test cases

File: tests/e2e/specs/customizer/footer-builder/margin.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/footer-builder/margin.test.js]: Added CamelCase

**Review by premanshup — APPROVED** [review]

**Pull request merged**